pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -255,10 +245,10 @@ func TestResourceLocation(t *testing.T) {
 
 	ctx := api.NewDefaultContext()
 	for _, tc := range testCases {
-		storage, _, _, fakeClient := newStorage(t)
+		storage, _, _, server := newStorage(t)
```

Comment:
So I've done that with other tests, I had chosen not to do that here in part because the data could collide and it's slightly orthogonal to the main purpose of the PR.  

I'd be happy to do a follow-up PR for some of these to cleanup, if that makes sense.